clients/java/signalr/src/main/java/com/microsoft/signalr/package-info.java

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+package com.microsoft.signalr;
```

Comment:
How does this improve the current doc? Do we need to put additional Javadocs in here to do that? The [current doc](https://docs.microsoft.com/en-us/java/api/com.microsoft.aspnet.signalr?view=aspnet-signalr-java) already has a list of Classes/Interfaces so I think what we're looking for is the ability to add a description...